Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -16,7 +16,15 @@ configuration:
   name: port
   required: true
   type: 0
-- defaultvalue: search index=notable | eval rule_name=if(isnull(rule_name),source,rule_name) | eval rule_title=if(isnull(rule_title),rule_name,rule_title) | `get_urgency` | `risk_correlation` | eval rule_description=if(isnull(rule_description),source,rule_description) | eval security_domain=if(isnull(security_domain),source,security_domain)
+- display: Trust any certificate (not secure)
```

Comment:
the order of the params is a bit messy, lets try to group them in some way that makes sense
the connection related checkboxes (proxy, insecure, python handler) should be together and below all other params